pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -432,18 +435,17 @@ func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templ
 
 	// LimitRange
 	cpuLimit := "500m"
-	memoryLimit := "512Mi"
+	memoryLimit := "750Mi"
 	memoryRequest := "64Mi"
 	cpuRequest := "10m"
 	if kind == "code" {
 		cpuLimit = "1000m"
 		cpuRequest = "60m"
 		memoryRequest = "307Mi"
+		memoryLimit = "512Mi"
 	}
```

Comment:
this test checks the current default tier limits right? if we would apply the same limits across all namespaces as part of https://issues.redhat.com/browse/CRT-940 this if kind == "code" block will be removed, right?

Reply:
If we apply the same limits for all NSs for all tiers then yes, that's correct.
But in this PR we apply the same limits only in the `test` tier which is not covered by this test.